.github/workflows/docker.yml

```diff
@@ -51,49 +56,665 @@ jobs:
+          cache-from: type=registry,ref=ghcr.io/noaa-gsl/dockerslurmcluster/node-cache-amd64:cache
+          cache-to: type=registry,ref=ghcr.io/noaa-gsl/dockerslurmcluster/node-cache-amd64:cache,mode=max
+          outputs: type=image,name=${{ env.REGISTRY_NODE_IMAGE }},push-by-digest=true,name-canonical=true,push=true
+      -
+        name: Export digest
+        run: |
+          mkdir -p /tmp/digests
+          digest="${{ steps.build.outputs.digest }}"
+          touch "/tmp/digests/${digest#sha256:}"
+      -
+        name: Upload digest
+        uses: actions/upload-artifact@v4
+        with:
+          name: node-digests-linux-amd64
+          path: /tmp/digests/*
+          if-no-files-found: error
+          retention-days: 1
+
+  merge-node:
+    runs-on: ubuntu-latest
+    needs:
+      - build-node-amd64
+      - build-node-arm64
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v4
+      -
+        name: Download digests
+        uses: actions/download-artifact@v4
+        with:
+          path: /tmp/digests
+          pattern: node-digests-*
+          merge-multiple: true
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY_NODE_IMAGE }}
+          tags: |
+            type=raw,value=latest
+          flavor: |
+            latest=true
+            prefix=
+            suffix=
+      -
+        name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          logout: false
+      -
+        name: Create manifest list and push
+        working-directory: /tmp/digests
+        run: |
+          docker buildx imagetools create $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
+            $(printf '${{ env.REGISTRY_NODE_IMAGE }}@sha256:%s ' *)
+      -
+        name: Inspect image
+        run: |
+          docker buildx imagetools inspect ${{ env.REGISTRY_NODE_IMAGE }}:${{ steps.meta.outputs.version }}
+
+#      -
+#        name: Build and push master
+#        uses: docker/build-push-action@v4
+#        with:
+#          context: ./master
+#          platforms: linux/amd64,linux/arm64
+#          file: ./master/Dockerfile
+#          push: true
+#          tags: noaagsl/slurm-master:latest
+#          cache-from: type=gha
+#          cache-to: type=gha,mode=max
+#
+#      -
+#        name: Build and push node
+#        uses: docker/build-push-action@v4
+#        with:
+#          context: ./node
+#          platforms: linux/amd64,linux/arm64
+#          file: ./node/Dockerfile
+#          push: true
+#          tags: noaagsl/slurm-node:latest
+#          cache-from: type=gha
+#          cache-to: type=gha,mode=max
```

Comment:
Do we need this for later? is this one of those circumstances where you will uncomment the code to run the commands in the future?

Reply:
No.  I forgot to remove it.